src/esn.calendar.libs/app/services/cached-event-source.js

```diff
@@ -197,15 +197,17 @@ require('./cal-ui-authorization-service.js');
       });
     }
 
-    function _handleDeclinedEvents(events) {
+    function _handleDeclinedEvents(calendar, events) {
+      const calendarOwner = calendar.isOwner(session.user._id);
```

Comment:
Does isOwner works when user A have has been delegated on user B calendar and user B have declined an event ?

Reply:
yes it is working , will put a demo with this senario

Reply:
**Demo with delegation:**
https://user-images.githubusercontent.com/43316837/105775518-362b2700-5f67-11eb-802e-59634270c5fe.mp4